Subject: [PATCH] fix gift
---
Index: raw_request.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/raw_request.go b/raw_request.go
--- a/raw_request.go	(revision fe82c1c5717c2bb716147bdbc2dde56bf9ce1b59)
+++ b/raw_request.go	(date 1749544700691)
@@ -84,6 +84,7 @@
 
 	r := apiResponse{}
 
+	//fmt.Println("======body=====", string(body))
 	errDecode := json.Unmarshal(body, &r)
 	if errDecode != nil {
 		return fmt.Errorf("error decode response body for method %s, %s, %w", method, body, errDecode)
Index: models/gift.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/models/gift.go b/models/gift.go
--- a/models/gift.go	(revision fe82c1c5717c2bb716147bdbc2dde56bf9ce1b59)
+++ b/models/gift.go	(date 1749544677678)
@@ -71,20 +71,37 @@
 
 // OwnedGiftRegular https://core.telegram.org/bots/api#ownedgiftregular
 type OwnedGiftRegular struct {
-	Type              OwnedGiftType `json:"type"`
-	Gift              Gift          `json:"gift"`
-	OwnedGiftID       string        `json:"owned_gift_id,omitempty"`
-	SenderUser        *User         `json:"sender_user,omitempty"`
-	SendDate          int           `json:"send_date"`
-	IsSaved           bool          `json:"is_saved,omitempty"`
-	CanBeTransferred  bool          `json:"can_be_transferred,omitempty"`
-	TransferStarCount int           `json:"transfer_star_count,omitempty"`
+	Type OwnedGiftType `json:"type"`
+	Gift Gift          `json:"gift"`
+
+	OwnedGiftID string `json:"owned_gift_id,omitempty"`
+	SenderUser  *User  `json:"sender_user,omitempty"`
+	SendDate    int    `json:"send_date"`
+	IsSaved     bool   `json:"is_saved,omitempty"`
+	
+	//CanBeTransferred  bool   `json:"can_be_transferred,omitempty"`
+	//TransferStarCount int `json:"transfer_star_count,omitempty"`
+
+	ConvertStarCount        int             `json:"convert_star_count,omitempty"`
+	PrepaidUpgradeStarCount int             `json:"prepaid_upgrade_star_count,omitempty"`
+	WasRefunded             bool            `json:"was_refunded,omitempty"`
+	CanBeUpgraded           bool            `json:"can_be_upgraded,omitempty"`
+	Text                    string          `json:"text,omitempty"`
+	Entities                []MessageEntity `json:"entities,omitempty"`
+	IsPrivate               bool            `json:"is_private,omitempty"`
 }
 
 // OwnedGiftUnique https://core.telegram.org/bots/api#ownedgiftunique
 type OwnedGiftUnique struct {
 	Type OwnedGiftType `json:"type"`
 	Gift UniqueGift    `json:"gift"`
+
+	OwnedGiftID       string `json:"owned_gift_id,omitempty"`
+	SenderUser        *User  `json:"sender_user,omitempty"`
+	SendDate          int    `json:"send_date"`
+	IsSaved           bool   `json:"is_saved,omitempty"`
+	CanBeTransferred  bool   `json:"can_be_transferred,omitempty"`
+	TransferStarCount int    `json:"transfer_star_count,omitempty"`
 }
 
 // OwnedGifts https://core.telegram.org/bots/api#ownedgifts
@@ -107,14 +124,14 @@
 // UniqueGiftModel https://core.telegram.org/bots/api#uniquegiftmodel
 type UniqueGiftModel struct {
 	Name           string  `json:"name"`
-	Sticker        Sticker `json:"sticker"`
+	Sticker        Sticker `json:"-"`
 	RarityPerMille int     `json:"rarity_per_mille"`
 }
 
 // UniqueGiftSymbol https://core.telegram.org/bots/api#uniquegiftsymbol
 type UniqueGiftSymbol struct {
 	Name           string  `json:"name"`
-	Sticker        Sticker `json:"sticker"`
+	Sticker        Sticker `json:"-"`
 	RarityPerMille int     `json:"rarity_per_mille"`
 }
 
